### ISSUE TITLE
Yarn: Workspaces can only be enabled in private projects

### ISSUE THREAD

**Issue by jackw** [wojtekmaj/enzyme-adapter-react-17#26]
Hello, I've tried to bump `@wojtekmaj/enzyme-adapter-react-17` to `0.6.3` and doing so results in yarn complaining about "Workspaces can only be enabled in private projects". 

I'm guessing it's caused by this project root `package.json` being used by the NPM module as opposed to the module living in a workspace and the root being a private module and managing all workspaces in the repository. There's a report about it [here](https://github.com/yarnpkg/yarn/issues/8580#issuecomment-847209711) in yarn repository.

**Comment by wojtekmaj** [comment]
Yarn 1.x?

**Comment by jackw** [comment]
Yep. We're still running Yarn 1.x and the project is already set to private and has its own workspaces. I was unaware that Yarn 2 / 3 removed this limitation. Maybe it's not worth the effort to support Yarn 1 but wanted to let you know in case you get other reports.

**Comment by drewlustro** [comment]
You can do this with `jq`

```sh
cat package.json | jq "to_entries | map(select(.key | test(\"workspaces\") | not)) | from_entries | ." > package.without-workspaces.json
```

**Comment by wojtekmaj** [comment]
I did a thing, but hit a roadblock in https://github.com/yarnpkg/berry/issues/3609 for automating it.
